go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/brahma-adshonor/micro_service
```

Comment:
suggest to remove the underscore mark in your go package name

Reply:
ok

Reply:
done